devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy

```diff
@@ -98,6 +98,8 @@ metadata {
 		fingerprint profileId: "0104", inClusters: "0000, 0004, 0003, 0006, 0019, 0002, 0009", manufacturer: "DAWON_DNS", model: "PM-S140R-ZB", deviceJoinName: "Dawon Switch" //DAWOS DNS In-Wall Switch PM-S140R-ZB
 		fingerprint profileId: "0104", inClusters: "0000, 0002, 0003, 0006", manufacturer: "DAWON_DNS", model: "PM-S150-ZB", deviceJoinName: "Dawon Switch" //DAWOS DNS In-Wall Switch PM-S150-ZB
 		fingerprint profileId: "0104", inClusters: "0000, 0002, 0003, 0006", manufacturer: "DAWON_DNS", model: "ST-S150-ZB", deviceJoinName: "Dawon Switch" //DAWOS DNS In-Wall Switch ST-S150-ZB
+        // Mixtile
+        fingerprint profileId: "0104", inClusters: "0000, 0004, 0005, 0006", outClusters: "0006, 0019", manufacturer: "Mixtile", model: "W10-WW80SR01", deviceJoinName: "Mixitle Relay Swtich" // Mixtile Relay Swtich
```

Comment:
Please correct spacing (use 1 tab instead of 4 spaces) and use 2 tabs here (as other fingerprints).
Please change deviceJoinName to ->  "Mixitle Switch"

Reply:
@dapbtxl Please let us know if the code will be changed according to above suggestions.

Reply:
Hi，I submitted new changes, but did not update them. Why?

Reply:
@dapbtxl This branch is quite outdated and there are conflicts with the master branch. Can You submit Your changes in a new pull request ?

Reply:
ok